app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -100,14 +101,29 @@ export class ReleaseNotes extends React.Component<IReleaseNotesProps, {}> {
       datePublished: `${oldestRelease.datePublished} to ${latestRelease.datePublished}`,
       enhancements: newReleases.flatMap(r => r.enhancements),
       bugfixes: newReleases.flatMap(r => r.bugfixes),
+      pretext: newReleases.flatMap(r => r.pretext),
       other: [],
       thankYous: [],
     }
   }
 
+  private getPretext = (pretext: ReadonlyArray<ReleaseNote>) => {
```

Comment:
Super minor, but I'd probably name it `renderPretext` to follow existing conventions.
```suggestion
  private renderPretext = (pretext: ReadonlyArray<ReleaseNote>) => {
```